landscaper/gardenlet/pkg/chart/test_common/gardenlet_chart.go

```diff
@@ -0,0 +1,742 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
Can you rename to `landscaper/gardenlet/pkg/chart/charttest/charttest.go`? Examples from the go library for common test utility pkgs - https://go.googlesource.com/go/+/go1.16.2/src/net/http/httptest and https://go.googlesource.com/go/+/go1.16.2/src/testing/iotest.